src/provider/[provider].md

```diff
@@ -6,30 +6,56 @@ title: Storage Provider Summary
 ```js
 import { LineGraph } from "../components/line-graph.js";
 import { getDateXDaysAgo } from "../utils/date-utils.js";
-const data = FileAttachment(`../data/${observable.params.provider}-spark-rsr-summary.json`).json();
+
+const rsrData = FileAttachment(`../data/${observable.params.provider}-spark-rsr-summary.json`).json();
+const ttfbData = FileAttachment(`../data/${observable.params.provider}-spark-retrieval-timings-summary.json`).json();
 ```
 
+
 <div class="hero">
   <body><a href="/"><img src="../media/spark-logomark-blue-with-bbox.png" alt="Spark Logo" width="300" /></a><body>
     <h2>Dashboard Beta</h2>
     <body><a href="https://filspark.com/dashboard" target="_blank" rel="noopener noreferrer">(Click here for Legacy Spark Grafana Dashboard)</a><body>
 </div>
 
-<h4>Storage Provider Spark RSR Summary</h4>
-<body>This section shows the storage provider Spark Retrieval Success Rate Score summary. You can adjust the date range. Records start on the 7th April 2024.</body>
-
-
 ```js
 const start = view(Inputs.date({label: "Start", value: getDateXDaysAgo(180) }));
 const end = view(Inputs.date({label: "End", value: getDateXDaysAgo(1) }));
 ```
 
 <h3>Stats for ${observable.params.provider}</h3>
 
-<div class="grid grid-cols" style="grid-auto-rows: 500px;">
+
+<div class="grid grid-cols-2">
+  <div>
+    <h4>Storage Provider Spark RSR Summary</h4>
+    <body>This section shows the storage provider Spark Retrieval Success Rate Score summary.</body>
+  </div>
+  <div>
+    <h4>Spark Miner Time To First Byte (TTFB)</h4>
+    <body>The section shows the median of all TTFB values for valid miner retrieval tasks.</body>
+  </div>
+</div>
+
+
+<div class="grid grid-cols-2" style="grid-auto-rows: 500px;">
   <div class="card">${
-    resize((width) => LineGraph(data, {width, title: "Retrieval Success Rate", start, end }))
+    resize((width) => LineGraph(rsrData, {width, title: "Retrieval Success Rate", start, end }))
   }</div>
+  <div class="card">
+      ${Plot.plot({
+      title: 'Time to First Byte (ms)',
+      x: { type: 'utc', ticks: 'day' }, // TODO: Change tick to month once we have more data
+      y: { grid: true },
```

Comment:
Can you please configure the chart so that the Y axis starts from `0` and the values are shown with zero decimal places?

Right now, it starts from at the minimum TTFB observed in the last few days, which makes it look like there is a huge change in TTFB, while in reality, the change was only ~3% on the screenshot below ([source](https://add-spark-ttfb-stats.spark-dashboard.pages.dev/provider/f03028412)):

<img width="706" alt="Screenshot 2025-01-16 at 15 41 35" src="https://github.com/user-attachments/assets/f238b1db-cda3-4370-a275-cdd9ba1e2897" />